tools/servicefabric_enums_to_strings_generator/servicefabric_enums_to_strings_generator.csproj

```diff
@@ -1,52 +1,53 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
```

Comment:
Wouldn't it be extremelly awesome if this would actually be a cmakelists part of the solution that is being built in this repo?

Task is fine.

Reply:
Extresmelly yes. Take created.https://msazure.visualstudio.com/One/_workitems/edit/15625359